client/client.go

```diff
@@ -1719,3 +1731,45 @@ func trimHTTPPrefix(str string) string {
 	str = strings.TrimPrefix(str, "https://")
 	return str
 }
+
+func (c *client) LoadGlobalConfig(ctx context.Context, names []string) ([]string, error) {
+	r, err := c.getClient().LoadGlobalConfig(ctx, &pdpb.LoadGlobalConfigRequest{Names: names})
+	if err != nil {
+		return nil, err
+	}
+	if r.GetOk() {
+		return r.GetValues(), nil
+	} else {
+		return nil, errors.New(fmt.Sprintf("result of load global config is not ok, maybe any of the key is not found: %v", names))
+	}
+}
+
+func (c *client) StoreGlobalConfig(ctx context.Context, name string, value string) error {
```

Comment:
Do we need metrics and opentracing support?

Reply:
in current stage we have very little global configs and none of them will be changed frequently